riocli/apply/resolver.py

```diff
@@ -139,7 +139,7 @@ def _list_functors(self, kind):
             'secret': self.client.list_secrets,
             "project": self.v2client.list_projects,
             "package": self.client.get_all_packages,
-            "staticroute": self.client.get_all_static_routes,
+            "staticroute": self.v2client.list_static_routes,
```

Comment:
@RomilShah There's one more function above this, `guid_functor`. Update that as well.